cmd/config/olm/olm.yml

```diff
@@ -0,0 +1,87 @@
+---
+global:
+  gc: {{.GC}}
+  gcMetrics: {{.GC_METRICS}}
+  measurements:
+    - name: jobSummary
+    - name: podLatency
+      thresholds:
+        - conditionType: Ready
+          metric: P99
+          threshold: {{.POD_READY_THRESHOLD}}
```

Comment:
I think this workload can go without a podLatency threshold, you're good to remove this line.